app/utils/deep_link/index.ts

```diff
@@ -88,21 +89,22 @@ export async function handleDeepLink(deepLinkUrl: string, intlShape?: IntlShape,
             }
             case DeepLink.Permalink: {
                 const deepLinkData = parsed.data as DeepLinkPermalink;
-                if (NavigationStore.hasModalsOpened() || ![Screens.HOME, Screens.CHANNEL, Screens.GLOBAL_THREADS, Screens.THREAD].includes(NavigationStore.getVisibleScreen())) {
+                if (NavigationStore.hasModalsOpened() ||
```

Comment:
0/5 add a few line breaks so this reads as
```
if (
    Condition1 ||
    Condition2
) {
```
instead of
```
if (Condition 1 ||
Condition2) {
```